.github/workflows/sims-protected.yml

```diff
@@ -0,0 +1,145 @@
+name: Sims-protected
+# Sims workflow runs multiple types of simulations (nondeterminism, import-export, after-import, multi-seed-short)
+# This workflow will ask for approval to run on all Pull Requests, if a .go, .mod or .sum file have been changed
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: protected
```

Comment:
no need to run simulations in the "protected" environment -- the reasoning here was that they were heavy, so better for them to be manually triggered.

I think enough just to restrict it via paths: https://github.com/crypto-org-chain/cronos/pull/493

Reply:
I removed the sims-protected.yml and add restrictions in sims.yml like this:
```go
on:
  push:
    branches:
      - main
      - release/**
    paths-ignore:
      - 'docs/**'
  pull-request:
    paths-ignore:
      - 'docs/**'
```
do you have other suggested paths to ignore?

Reply:
I'm not sure... Also I'm thinking that sims may be even heavier than other GH actions -- perhaps just adding a "merge" queue: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue
so simulations are executed on a PR just before it's merged to main?
@yihuang

Reply:
but sometimes we also want to trigger it early just to see if it fails?

Reply:
or should we add more restrictions by adding more paths to the `paths-ignore` option?

Reply:
I think the chain-main's approach to controlling using the comments is pretty good, if we can make the code simpler.

Reply:
one thing is if the code comment one can be specified against a team instead of individual accounts